File: client/src/app/_dtos/admin/cvsAvailableDto.ts
import { IUserProfession } from "src/app/_models/params/Admin/userProfession";

export interface ICvsAvailableDto {
     candAssessmentId: number;
     checked: boolean;
     candidateId: number;
     mobileNo: string;
     gender: string;
     applicationNo: number;
     fullName: string;
     city: string;
     orderCategoryRef: string;
     orderItemId: number;
     gradeAssessed: string;
     assessedOn: Date;
     userProfessions: IUserProfession[];
}

export class CvsAvailableDto implements ICvsAvailableDto {
     candAssessmentId = 0;
     checked=false;
     candidateId = 0;
     mobileNo = '';
     gender='male';
     applicationNo = 0;
     fullName = '';
     city= '';
     orderCategoryRef = '';
     orderItemId=0;
     gradeAssessed='';
     assessedOn=new Date();
     userProfessions: IUserProfession[]=[];
}

File: api/DTOs/HR/CvsAvailableDto.cs
using api.Entities.HR;

namespace api.DTOs.HR
{
    public class cvsAvailableDto
    {
        public int CandAssessmentId { get; set; }
        public int CandidateId { get; set; }
        public bool  Checked { get; set; }
        public string Gender { get; set; }
        public int ApplicationNo { get; set; }
        public string FullName {get; set;}
        public string City {get; set;}
        public string OrderCategoryRef { get; set; }
        public int OrderItemId { get; set; }
        public string GradeAssessed { get; set; }
        public DateTime AssessedOn { get; set; }
        public ICollection<UserProfession> userProfessions { get; set; }
    }
}